lib/dialects/mssql/connection-manager.js

```diff
@@ -57,6 +59,10 @@ class ConnectionManager extends AbstractConnectionManager {
         delete connectionConfig.options.port;
       }
 
+      if (config.dialectOptions.encrypt && _.isBoolean(config.dialectOptions.encrypt)) {
```

Comment:
`connectionConfig.encrypt` will have to effect, as only `connectionConfig.options.encrypt` will be used, which can be overriden by setting `dialectOptions` in sequelize config.

Remove this change